.github/CODEOWNERS

```
@@ -0,0 +1 @@
+* @JourneyBook
```

```suggestion
* @collab-community/journey-book 
```
This may work not sure.

---

even what i did may not work, this is better, merging but we gotta test.